spec/models/system_admin/funding_upload_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module SystemAdmin
+  describe FundingUpload do
+    subject { build(:funding_upload) }
+
+    it { is_expected.to be_valid }
+
+    describe "scopes" do
+      describe ".recently_processed_upload_for" do
+        let!(:older_processed_funding_upload) { create(:funding_upload, :processed, :lead_school_trainee_summaries, created_at: 1.day.ago) }
+        let!(:recently_processed_funding_upload) { create(:funding_upload, :lead_school_trainee_summaries, :processed) }
+        let!(:pending_funding_upload) { create(:funding_upload) }
+        let!(:failed_funding_upload) { create(:funding_upload, :failed) }
```

Comment:
is it worth making the `funding_type` explicit in the last two of these as well, by adding the `:lead_school_trainee_summaries` trait?